distribution/changelog.txt

```diff
@@ -1,5 +1,6 @@
 0.4.5 (in development)
 ------------------------------------------------------------------------
+- Feature: [#19446] Add new color options to color dropdown.
```

Comment:
Should have been `colour` - we use British spelling throughout.

Reply:
Gah, figures I'd slipped up at some point. Will make a quick fix

Reply:
No need to worry, I already created a PR that also fixed some other things. I just figured it would be a good idea to point it out anyway.